src/rabbit_quorum_queue.erl

```diff
@@ -92,6 +99,9 @@
         ]).
 
 %%----------------------------------------------------------------------------
+set_data_dir() ->
+    {ok, BaseDir} = application:get_env(mnesia, dir),
+    application:set_env(ra, data_dir, filename:join(BaseDir, "quorum")).
```

Comment:
Now that https://github.com/rabbitmq/ra/pull/37 is merged, we can check that `ra`, `data_dir` enviromnent variable is set and only use the mnesia subdirectory if it's not set before.
This will allow setting a separate ra data dir via config files.